project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -23,13 +24,14 @@
 
 @WebServlet("/create-event")
 public class EventCreationServlet extends HttpServlet {
-  private static final DatabaseWrapper dbWrapper = new DatabaseWrapper("step-188-instance", "event-organizer-db");
-
   /** Returns event details from database */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String eventId = request.getParameter("eventId");
+
+    DatabaseWrapper dbWrapper = new DatabaseWrapper(new WrapperDatabaseService());
```

Comment:
Could this use the same dbWrapper as the doPost() in line 63? like private static final or not?

Reply:
We cannot because this is called only when the servlet is initialized (runs only once). So if we initialize using private static final and close it in doGet(), then the next time we do a GET, the connection that came from when the servlet is initialized will be closed.

Reply:
Ok makes sense ty for the clarification!